federation-2/router-bridge/js-src/plan_worker.ts

```diff
@@ -191,10 +191,21 @@ async function run() {
         }
       } catch (e) {
         logger.warn(`an error happened in the worker runtime ${e}\n`);
+
+        const unexpectedError = {
+          name: e.name || "unknown",
+          message: e.message || "",
+          extensions: {
+            code: "QUERY_PLANNING_FAILED",
+            exception: {
+              stacktrace: [e.toString().split(/\n/)],
```

Comment:
I know this is still work in progress, but note that this isn't right and will result in an array within an array.  Removing the outside `[`/`]` would be correct.